src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -609,10 +609,10 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
       scheduleTransition(WorkflowFailed)
       stay()
     case Event(message @ AsyncFailure(t), _) =>
-      // This is the unusual combination of debug + throwable logging since the expectation is that this will eventually
+      // This is the unusual combination of warn + throwable logging since the expectation is that this will eventually
       // be logged in the case above as an error, but if for some weird reason this actor never ends up in that
```

Comment:
Currently we are getting no messages at all and this might be the cause.  We can adjust based on what happens when this is deployed.  :smile: